styles/layout.css

```diff
@@ -0,0 +1,9 @@
+@media only and (min-width: 640px) {
+	.nav ul {
+		display: block;
+	}
+}
+
+.nav ul {
+	display: inline-block;
+}
```

Comment:
This CSS selector is applying to all classes called 'nav'. But you don't have any such classes in your HTML. You currently have a <nav> element with 'main-nav' as its class.  

Also, I think the child element you want to specify is 'li' not 'ul'